src/Integration.Vsix.UnitTests/Suppression/DelegateInjectorTests.cs

```diff
@@ -128,33 +164,46 @@ class SonarAnalysisContext
   }
 }";
             // Arrange
-            Assembly asm = CreateAndLoadAssembly(SonarAnalyzerAssemblyName, "101.0", SourceCodeWithBadStaticProperty);
+            Assembly asm = CreateAndLoadAssembly(SonarAnalyzerAssemblyName, "104.0", SourceCodeWithBadStaticProperty);
             AssertSuppressionPropertyIsNotSet(asm);
 
             // Act - should not error
-            using (new DelegateInjector(diag => true))
+            using (new AssertIgnoreScope()) // Missing output window service
```

Comment:
If there is an error we write to the output window. However, the service provider isn't set up to provide a dummy output window, so our _VsShellUtils_ class will _Debug.Assert_ which would hang the test. The _AssertIgnoreScope_ suppresses the _Debug.Assert_.